assets/javascript/script.js

```diff
@@ -37,37 +37,46 @@ $(document).ready(function () {
             console.log(results);
 
             for (var i = 0; i < results.length; i++) {
-
                 let recipeDiv = $("#dinner-table");
                 let recipeTitle = results[i].title;
                 let recipeID = results[i].id;
+                let SITE_BASE = "https://spoonacular.com/recipes/";
```

Comment:
You can move this outside the for loop

Reply:
I left it in the for loop because it's part of the build up of the hyperlink URL.  It is meant for readability.  There's no reason to have it outside.